pkg/engine2/testdata/ecs_rds.err.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "error": {
+      "chain": [
+        "required property Id is not set on resource aws:subnet:vpc-0:subnet-0"
```

Comment:
`Id` is one of the "required + deploy time" variables. Those should probably not cause validation errors.

Reply:
yeah we have a card for this, so we might want to pick it up as a part of this workstream